DataFormats/Common/interface/MapOfVectors.h

```diff
@@ -134,6 +134,10 @@ namespace edm {
       return *this;
     }
 
+    MapOfVectors(const MapOfVectors&) = default;
```

Comment:
With this (explicit definitions) approach, please add move constructor and assignment as well. Alternatively, it seems to me the explicit copy assignment operator could be removed and left to be generated by the compiler, or did I miss something?

Reply:
All the data members are just `std::vector`s, so the (compiler-generated) default assignment operator should provide the same behavior as the current explicitly defined assignment operator.